app/views/groups/members/_access_level.html.erb

```diff
@@ -9,8 +9,8 @@
         id: "member-#{member.id}-access-level-select",
         value: member.access_level,
         onchange: "this.form.requestSubmit();",
-        "aria-label": "Access Level"
-      }
+        "aria-label": "Access Level",
```

Comment:
We should localize the aria-label text here and the projects one as well

Reply:
5f7e635939d0be815d43b61f272ea197c105c9e0